ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/phase0/BeaconBlockBodyBuilderPhase0.java

```diff
@@ -111,6 +112,13 @@ public BeaconBlockBodyBuilder executionPayload(
     return this;
   }
 
+  @Override
+  public BeaconBlockBodyBuilder executionPayloadHeader(
+      Supplier<ExecutionPayloadHeader> executionPayloadHeaderSupplier) {
+    // No execution payload in phase 0
+    return null;
```

Comment:
should return `this`
```suggestion
    return this;
```

Reply:
oops